packages/postgraphile-core/__tests__/integration/schema/omit-rename.omitstuff.test.js

```diff
@@ -56,13 +56,25 @@ comment on table d.films is E'@omit read,all,update,create,delete,many';
 );
 
 test(
-  "omit and omit many on table",
+  "omit and omit everything on table",
   core.test(
     ["d"],
     {},
     `
 comment on table d.films is E'@omit *';
 comment on table d.tv_shows is E'@omit';
+`
+  )
+);
+
+test(
+  "omit many on constraint and table",
+  core.test(
+    ["d"],
+    {},
+    `
+comment on constraint post_author_id_fkey on d.post is E'@omit many\n@fieldName author';
+comment on constraint tv_shows_studio_id_fkey on d.tv_shows is E'@omit many';
 comment on table d.tv_episodes is E'@omit many';
```

Comment:
I've moved this line into the new test, in the previous "omit and omit many on table" test it had no effect - the only table referencing `tv_episodes` was `tv_shows`, which was omitted entirely.